mteb/models/arctic_models.py

```diff
@@ -109,11 +109,14 @@
         # splits not specified to assuming everything
         # in MTEB
         "NQ": ["test"],
+        "NQ-NL": ["test"],  # translated from NQ (not trained on)
         "NQHardNegatives": ["test"],
         "HotPotQA": ["test"],
         "HotPotQAHardNegatives": ["test"],
         "HotPotQA-PL": ["test"],  # translated from hotpotQA (not trained on)
+        "HotpotQA-NL": ["test"],  # translated from hotpotQA (not trained on)
```

Comment:
@KennethEnevoldsen if these are "not trained on", should we still keep these? Personally I find these very confusing.

Reply:
I think translating a dataset and training on it should still lead to a non-zero-shot on the benchmark - these are just to annotate that. We could "link" the tasks and update leaderboard code (but currently that is not how it is done)

Reply:
Got it, makes sense, thanks. The "not trained on" part was confusing for me. Maybe it could have said something closer to "trained on translation" in the future?